core/MODULE.bazel

```
@@ -5,5 +5,3 @@ module(
 
 bazel_dep(name = "platforms", version = "0.0.4")
 bazel_dep(name = "bazel_skylib", version = "1.0.3")
```

```suggestion
bazel_dep(name = "bazel_skylib", version = "1.0.3")
bazel_dep(name = "rules_sh", version = "0.3.0", dev_dependency = True)
```
IIUC `stardoc` is indeed no longer a dependency of `rules_nixpkgs_core`. However, `rules_sh` is still needed for the tests, but, not for regular usage. In that case it should be marked as a dev-dependency.